blank-canvas-blocks/experimental-theme.json

```diff
@@ -413,6 +414,11 @@
 		"core/separator": {
 			"color": {
 				"background": "var(--wp--custom--color--foreground)"
+			},
+			"border": {
+				"color": "currentColor",
+				"style": "solid",
+				"width": "0 0 1px 0"
```

Comment:
The border definitions work fine, but the background is not doing really anything. It's the color property that's picked up by currentColor on the border. You can test this by changing colors on the editor, or try and set `"background": "var(--wp--custom--color--foreground)" `to red so you can see that it doesn't really work.